x/leverage/keeper/keeper_test.go

```diff
@@ -128,6 +147,314 @@ func (suite *IntegrationTestSuite) TestWithdrawAsset_Valid() {
 	suite.Require().Equal(int64(0), uTokenBalance.Amount.Int64())
 }
 
+// Helper function: Initialize the common starting scenario from which borrow and repay tests stem:
+func (suite *IntegrationTestSuite) initBorrowScenario() (lender, bum sdk.AccAddress) {
+	app, ctx := suite.app, suite.ctx
```

Comment:
nit: try to add newlines after logical groupings of calls. This method is way too difficult to read.